data/data/agent/systemd/units/node-zero.service

```diff
@@ -1,7 +1,8 @@
 [Unit]
```

Comment:
Shouldn't node-zero.service wait for /etc/assisted/rendezvous-host.env to be present?

Reply:
It's done within the script (just merged 4 hours ago): #7068

Reply:
Ah thanks for reminding me.

Reply:
So, I didn't have the 7608 PR included but I set node-zero.service to be 
`After=network-online.target load-config-iso@.service`
Since load-config-iso@.service writes /etc/assisted/rendezvous-host.env it was there when node-zero.service ran